README.md

```diff
@@ -240,9 +250,9 @@ By the end of this lesson. You should be able to set up two separate servers tha
 
 * **Question:** Why did the number of posts not change when you were redirected back to the `/users` route?
 
-* **Your Answer:** Whenever we modify our data with a Create, Update, or Delete, we have a few options on how to make our frontend reflect those changes. What options can you think of?
+* **Your Answer: There was not an action to change the state of the container 
 
-* **Question:**
+* **Question: Whenever we modify our data with a Create, Update, or Delete, we have a few options on how to make our frontend reflect those changes. What options can you think of? - Can refetch the date, remove a post from state and update the front end, 
```

Comment:
We could re pull from the server, or if we trust the data was added to the db we could modify state directly